identity/webapp/src/frontend/MyAccount/ChangePassword.tsx

```diff
@@ -40,6 +41,10 @@ export const ChangePassword: React.FC<ChangeDetailsModalContentProps> = ({ onCom
         setError('password', { type: 'manual', message: 'Incorrect password.' });
         break;
       }
+      case RequestDeleteError.BRUTE_FORCE_BLOCKED: {
+        setError('password', { type: 'manual', message: 'Your account has been blocked after multiple consecutive login attempts.' });
```

Comment:
As above, this error isn't necessarily caused by the password field.

Reply:
Fixed.